src/pages/Login.tsx

```diff
@@ -0,0 +1,199 @@
+// import React from 'react';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import login_image from '../assets/login-image.webp'
+import intercars_logo from '../assets/ic_new_logo.jpg'
+import { useForm, Controller, SubmitHandler } from "react-hook-form"
+// import { IconButton } from '@mui/material';
+// import InputAdornment from '@mui/material/InputAdornment';
+import Visibility from '@mui/icons-material/Visibility';
+import VisibilityOff from '@mui/icons-material/VisibilityOff';
+import { yupResolver } from "@hookform/resolvers/yup"
+import * as yup from "yup"
+import { useTranslation } from 'react-i18next';
+import ShowHideFunctionality from '@/components/shared/ShowHideFunctionality';
+
+
+export default function Login() {
+
+
+  
+
+
+    const { t: translate } = useTranslation()
+    const schema = yup
+        .object({
+            email: yup.string().required(translate('errors.email.required')).email(translate('errors.email.invalid')),
+            password: yup.string().required(translate('errors.password.required')).min(4, translate('errors.password.short')).max(10, translate('errors.password.long'))
+                .matches(/[0-9]/, translate('errors.password.digit'))
+                .matches(/[!@#$%^&*]/, translate('errors.password.specialCharacter'))
+        })
+        .required()
+
+
+    interface LoginSchema extends yup.InferType<typeof schema> {
+        email: string;
+        password: string;
+    }
+
+    // const [showPassword, setShowPassword] = React.useState(false);
+    const { control, handleSubmit, formState: { errors } } = useForm<LoginSchema>({
+        resolver: yupResolver(schema),
+    })
+
+    const onSubmit: SubmitHandler<LoginSchema> = (data) => console.log(data)
+
+    // const handleClickShowPassword = () => setShowPassword((show) => !show)
+
+    // const handleMouseDownPassword = (event: React.MouseEvent<HTMLButtonElement>) => {
+    //     event.preventDefault()
+    // }
+
+    const formHasErrors = () => {
+        return Object.keys(errors).length > 0
+    }
+
+
+    return (
+        <Grid container component="main" sx={{ height: '100vh' }}>
+            <CssBaseline />
```

Comment:
remove this, its imported inside `App.tsx` so its global

Reply:
Remove <CssBaseline />